source/dotnet/Library/AdaptiveCards/AdaptiveTypedElement.cs

```diff
@@ -4,6 +4,12 @@
 
 namespace AdaptiveCards
 {
+    //[Serializable]
+    public delegate void TypedEventHandler<in TSender, in TEventArgs>(
```

Comment:
>TypedEventHandler [](start = 25, length = 17)

Not sure what the purpose of this is, but does it have chances for conflict with Windows.Foundation.TypedEventHandler? [https://docs.microsoft.com/en-us/uwp/api/windows.foundation.typedeventhandler](https://docs.microsoft.com/en-us/uwp/api/windows.foundation.typedeventhandler)

Reply:
It's the same thing, but isn't part of net standard so I had to reuse the concept

Reply:
That's too bad, makes sense